### ISSUE TITLE
A.1.2 Extended behaviour

### ISSUE THREAD

**Issue by jensscheerlinck** [inspire-eu-validation/ats-discovery-service#10]
_This issue has been extracted from the issue list on:https://ies-svn.jrc.ec.europa.eu/issues/2685_
# Comment

What is this for?

**Comment by jensscheerlinck** [comment]
# Proposed Resolution

Remove test case because of redundancy:
ôThe test is an abstraction of all the tests in this Abstract Test Suiteö
Depends on issue #29

**Comment by PeterParslow** [comment]
Duplicate of #29.
